src/website/app/demos/layout/examples/flex-item/grow.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import DemoLayout from '../../components/DemoLayout';
+import FlexItem from '../../components/FlexItem';
+import Flex from '../../components/Flex';
+
+export default {
+  id: 'grow',
+  title: 'Grow',
+  description: `By default, FlexItems will not expand to fill their container
+(\`grow={0}\`). Setting the \`grow\` prop to a positive number allows you to
+determine the rate at which they will expand, relative to their siblings.
+
+For the simplest case where each item expands at the same rate, set \`grow={1}\`
+on all FlexItems, as in the first row below. If different values are set, then
+they grow proportionally. E.g., The total of values for the items in the third
```

Comment:
I think it's a little strange to have "E.g." capitalized and "The" also capitalized. Perhaps just spell out "For example?"